modules/images/src/lib/parsers/parse-image.js

```diff
@@ -11,9 +11,7 @@ export default async function parseImage(arrayBuffer, options, context) {
   options = options || {};
 
   const {url} = context || {};
-  if (url && /\.svg((\?|#).*)?$/.test(url)) {
-    // eslint-disable-next-line
-    console.warn('@loaders.gl/images: SVG parsing needs to be fixed for 2.0');
+  if (url && /^data:image\/svg\+xml|\.svg((\?|#).*)?$/.test(url)) {
```

Comment:
Nit: I would prefer splitting this into two named regexps, one for the file extension and one for the data URL pattern.

As it stands this is hard to read and it is a problematic part of the code as we are doing a second URL extension sniffing/type detection outside of loaders.gl system. 

We can help a little by making the code as clear as possible and maybe mentioning in a comment that we are taking these "liberties".

Reply:
+1